projects/plugins/crm/src/js/components/automations-admin/components/step-config/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { Step } from 'crm/state/automations-admin/types';
+import { AttributeConfig } from '../attribute-config';
+import styles from './styles.module.scss';
+
+type StepConfigProps = {
+	workflowId: number;
+	step: Step;
+};
+
+export const StepConfig: React.FC< StepConfigProps > = ( { workflowId, step } ) => {
+	const attributes = Object.values( step.attribute_definitions ).map( definition => ( {
+		definition,
+		value: step.attributes[ definition.slug ],
+	} ) );
+
+	return (
+		<>
+			<div className={ styles.title }>{ step.title }</div>
```

Comment:
I think this is supposed to be "bold". I know the fonts are different from Figma, so "400" in weight doesn't necessarily translate into "bold".

I think the font size is supposed to be 16px as well